SPECS-EXTENDED/exiv2/exiv2.spec

```
@@ -44,38 +34,31 @@ A command line utility to access image metadata, allowing one to:
 * extract, insert and delete Exif metadata (including thumbnails),
   Iptc metadata and Jpeg comments
 
-%package devel
-Summary: Header files, libraries and development documentation for %{name}
-Requires: %{name}-libs%{?_isa} = %{version}-%{release}
+%package      devel
+Summary:      Header files, libraries and development documentation for %{name}
+Requires:     %{name}-libs%{?_isa} = %{version}-%{release}
 # FIXME/TODO: probably overlinking --rex
 # exiv2/exiv2Config.cmake:  INTERFACE_LINK_LIBRARIES "/usr/lib64/libexpat.so"
-Requires: expat-devel%{?_isa} 
-%description devel
+Requires:     expat-devel%{?_isa} 
+%description  devel
 %{summary}.
 
-%package libs
-Summary: Exif and Iptc metadata manipulation library
-# not strictly required, but convenient and expected
-%if 0%{?rhel} && 0%{?rhel} <= 7
-Requires: %{name} = %{version}-%{release}
-%else
-Recommends: %{name} = %{version}-%{release}
-%endif
-%description libs
+%package      libs
+Summary:      Exif and Iptc metadata manipulation library
+Recommends:   %{name} = %{version}-%{release}
+%description  libs
 A C++ library to access image metadata, supporting full read and write access
 to the Exif and Iptc metadata, Exif MakerNote support, extract and delete
 methods for Exif thumbnails, classes to access Ifd and so on.
 
-%package doc
-Summary: Api documentation for %{name}
-BuildArch: noarch
-%description doc
+%package      doc
+Summary:      Api documentation for %{name}
+BuildArch:    noarch
+%description  doc
 %{summary}.
 
-
 %prep
-%autosetup -n %{name}-%{version}-%{?beta}%{!?beta:Source} -p1
-
+%autosetup -n %{name}-%{version} -p1
```

It it's `%{name}-%{version}`, then I believe that's the default and we can completely the `-n` argument:
```suggestion
%autosetup -p1
```